### ISSUE TITLE
Date not show in blog posts

### ISSUE THREAD

**Issue by edlgg** [serverless/blog#823]
It would be very helpful if the date of creation was shown at the top in each blog post.

**Comment by montao** [comment]
> 
> 
> It would be very helpful if the date of creation was shown at the top in each blog post.

Yes. The date in the file name and in the meta data of the file contents, in the header of the text as a comment. So either parse the file name or get it from the meta data in the file. The actual links to the posts do display the date of creation. But I think that the format should be e.g. 2019-08-15 because 150819 could be misunderstood as 2015-08-19 for many of us. I will try and create a PR for it if noone else does.
